mock/py/mockbuild/config.py

```diff
@@ -687,9 +741,12 @@ def load_config(config_path, name, uidManager, version, pkg_python_dir):
             chroot_cfg_path = '%s/%s.cfg' % (config_path, name)
     config_opts['config_file'] = chroot_cfg_path
 
-    cfg = os.path.join(config_path, 'site-defaults.cfg')
-    do_update_config(log, config_opts, cfg, uidManager, name)
+    # load the global config files
+    for cfg_file in ["site-defaults.cfg", "chroot-aliases.cfg"]:
```

Comment:
I wonder why we need new config for that? This can be in default config (when done in upstream) or in site-defaults or .config/mock.cfg when user wants define their own.

Reply:
Because site-defaults.cfg is empty upstream, and users put there their stuff.
If we add something there it will not be delivered to many of our users.

Reply:
Because "noreplace"

Reply:
I eventually moved this to /usr/lib/mock/site-defaults.cfg as we anyway plan to move
everything there in v3.0+.

Reply:
The config files are architecture independent (we ship them in a _noarch_ package, so `/usr/share` is much more sensible).

Reply:
I tried to mimic how this is done in rpm/systemd (note it is not %_libdir).

Reply:
The reasons why systemd used `/usr/lib` don't apply here, and rpm's usage of `/usr/lib` is an ugly accident.

Reply:
Citation needed!

Reply:
I'd have to dig up the old references, but the reason systemd uses `/usr/lib` instead of `/usr/share` was that Debian wasn't willing to put a top level `/share` directory for its immutable files, so `/lib` is where it went. RPM's usage of `/usr/lib` for vendor/static configuration is pretty much an accident as it was originally for helper binaries and scripts (which was supposed to to be in `/usr/libexec`, but back in 1995, nobody was sure of anything yet). The path got co-opted later for other things. One day, it's on my TODO to actually fix that mess in RPM, but it'll be a while.

Reply:
> One day, it's on my TODO to actually fix that mess in RPM, but it'll be a while.

It would be nice if this was tracked somewhere else than your personal TODO list,
because it's worth seeing the feedback from the community about this.

I tend to fall back to keep the stuff in /etc till we have some authoritative answer
on how to handle this scenario :-(